components/gitpod-db/src/typeorm/migration/helper/helper.ts

```diff
@@ -25,7 +25,7 @@ export async function indexExists(queryRunner: QueryRunner, tableName: string, i
                 AND table_name = '${tableName}'
                 AND index_name = '${indexName}'`
     );
-    return Number.parseInt(countResult[0].cnt) === 1;
+    return Number.parseInt(countResult[0].cnt) > 0; // for composite indexes this seems to return the number of columns involved
```

Comment:
Nit: Maybe it's safer to specify base 10?

```suggestion
    return Number.parseInt(countResult[0].cnt, 10) > 0; // for composite indexes this seems to return the number of columns involved
```